tensorflow/python/ops/array_ops.py

```diff
@@ -322,18 +322,14 @@ def setdiff1d(x, y, index_dtype=dtypes.int32, name=None):
   out[i] = x[idx[i]] for i in [0, 1, ..., len(out) - 1]
   ```
 
-  For example, given this input:
-  ```python
-  x = [1, 2, 3, 4, 5, 6]
-  y = [1, 3, 5]
-  ```
-
-  This operation would return:
+  #### EXAMPLE:
 
-  ```python
-  out ==> [2, 4, 6]
-  idx ==> [1, 3, 5]
-  ```
+  >>> x = [1, 2, 3, 4, 5, 6]
+  >>> y = [1, 3, 5]
+  >>> result = tf.setdiff1d(x,y)
+  >>> sess = tf.compat.v1.Session()
```

Comment:
I don't think we want `Session` in the docs anymore, since it has been removed.

Can you rewrite this in TF2 format?